plugins/clipboard/plugin.js

```diff
@@ -2539,6 +2583,227 @@
 			return undefined;
 		}
 	};
+
+	/**
+	 * Fallback dataTransfer object which is used together with {@link CKEDITOR.plugins.clipboard.dataTransfer}
+	 * for browsers supporting Clipboard API, but not supporting custom MIME types (Edge 16+).
```

Comment:
Maybe we'll link `Edge 16+` to https://github.com/ckeditor/ckeditor-dev/issues/962?